fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -61,7 +61,7 @@ object FaciaContent {
 
     CuratedContent(
       content,
-      trailMetaData.headline.orElse(contentFields.get("headline")),
```

Comment:
Yep, that's exactly what I'm doing in MAPI now, be good to get that into the lib so we don't have to do the same thing over and over.